UNRELEASED.md

```diff
@@ -29,4 +30,9 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Code quality
 
+- Banner, DropZone, and ResourceList’s icons were moved to polaris-icons ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
+
 ### Deprecations
+
+- Passing a React Element into the Icon component is now deprecated ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

Comment:
Should we mention that passing a react component instead would work?

Reply:
There's a link to the PR and the warning explains what to do too, I think it's fine to leave as is but if you want to make a suggestion like Tim did then sure I'll commit it

Reply:
```suggestion
- Passing a React Element into the Icon component is now deprecated in favor of passing a React Component ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```